src/main.rs

```diff
@@ -15,6 +20,32 @@ fn parse_matches() -> ArgMatches<'static> {
                 .default_value("default")
                 .help("namespace to deploy test cluster"),
         )
+        .arg(
+            Arg::with_name("deploy_method")
+                .long("deploy-method")
+                .takes_value(true)
+                .possible_values(&["local", "tar", "skip"])
+                .default_value("local")
+                .help("Deploy method. tar, local, skip. [default: local]"),
+        )
+        .arg(
+            Arg::with_name("local-path")
```

Comment:
Be careful here, you're using it correctly in the code, but it's easy to make a mistake with having some args using snake case and others with kebab case.

For consistency, please update to snake case here and where it's extracted